index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang='cs'>
+	<head>
+    	<title>index</title>
+    	<meta charset='utf-8'>
+    	<meta name='description' content=''>
+    	<meta name='keywords' content=''>
+    	<meta name='author' content=''>
+    	<meta name='robots' content='all'>
```

Comment:
These
```html
<meta name='' content=''>
```
shouldn't be there, at least not empty. You should put them here, when you want to make this web public.